frontend/plugin_init.rb

```diff
@@ -2,19 +2,13 @@
 
   # check configuration
   if AppConfig.has_key?(:user_defined_in_basic)
```

Comment:
I took out the hard-coded list of record types but kept the warning about the user_defined schema.  I think it should now work for future record types too, so I didn't want to limit it artificially...